src/components/autocomplete.ts

```diff
@@ -69,10 +164,10 @@ const getCompletionIcon = (kind: CompletionItemKind) =>
 
 // TODO: move to common place. used in other places like signature-hint
 const parseDocs = async (
-  docs?: string | MarkdownString
```

Comment:
signature hints can contain Markdown, can't they?

Reply:
Yes, but I think the markdown is just going to be in regular strings. If I understand correctly, `MarkdownString` is a type that is only provided from VSCode completions . . . although I could be wrong.